src/index.js

```diff
@@ -1,8 +1,12 @@
 import React from 'react';
 import MapScreen from './Components/MapScreen/MapScreen';
+import SplashScreen from './Components/SplashScreen/SplashScreen';
```

Comment:
Doesn't appear to be getting used here.

Reply:
Ha! Yes. I was still finding a way to add splash screens other than adding them separately to android and ios. And I forgot to remove this import.